src/utils/yelpFusionApiUtils.ts

```diff
@@ -19,18 +19,18 @@ export class YelpFusionApiUtils {
 				latitude: geocode.lat,
 				longitude: geocode.lng,
 			})
-			.then((response) => response.jsonBody.businesses[0].id)
-			.then((id) => this.yelpClient.business(id))
-			.then((businessResult) =>
+			.then((response: any) => response.jsonBody.businesses[0].id)
+			.then((id: any) => this.yelpClient.business(id))
```

Comment:
```suggestion
			.then((id: string) => this.yelpClient.business(id))
```

Or is it `string | undefined`?